src/components/FirstTimeUserFlow/RoommatePrefForm.js

```diff
@@ -0,0 +1,107 @@
+import * as React from 'react';
+import { useState, useEffect } from 'react';
+import Grid from '@mui/material/Grid';
+import Typography from '@mui/material/Typography';
+import TextField from '@mui/material/TextField';
+import {
+	FormControlLabel,
+	FormControl,
+	FormLabel,
+	FormGroup,
+	RadioGroup,
+	Radio,
+	Select,
+	MenuItem,
+	Box,
+	Chip,
+	Checkbox,
+} from '@mui/material/';
+
+const dislikeOptions = ['Smoking', 'Alcohol', 'Pets', 'Couples', 'Children', 'Parties'];
+const hobbies = [
+	'Reading',
+	'Watching TV',
+	'Family Time',
+	'Movies',
+	'Fishing',
+	'Computer',
+	'Gardening',
+	'Renting Movies',
```

Comment:
```suggestion
	'Exercise',
```

Replacing since Movies is already in this list, that was clearly a really low quality list of hobbies I found if it has "Movies" *and* "Renting Movies" and they're only 5 elements away lol